Makefile

```diff
@@ -20,14 +19,6 @@ worker: files/install-deps-worker.yaml files/recipe-worker.yaml
 	$(CONTAINER_ENGINE) pull $(BASE_IMAGE)
 	$(CONTAINER_ENGINE) build --rm -t $(WORKER_IMAGE) -f files/docker/Dockerfile.worker --build-arg SOURCE_BRANCH=$(SOURCE_BRANCH) .
 
-# This is for cases when you want to deploy into production and don't want to wait for dockerhub
-# Make sure you have latest docker.io/usercont/packit:prod prior to running this
-worker-prod: files/install-deps-worker.yaml files/recipe-worker.yaml
-	$(CONTAINER_ENGINE) build --rm -t $(WORKER_IMAGE_PROD) -f files/docker/Dockerfile.worker --build-arg SOURCE_BRANCH=$(SOURCE_BRANCH) .
-
-worker-prod-push: worker-prod
-	$(CONTAINER_ENGINE) push $(WORKER_IMAGE_PROD)
```

Comment:
I'd say it is good we remove this and build only with actions/quay which are consistent, clearly defined and traceable. Builds and pushes from our laptops are not.